google-cloud-language/.yardopts

```diff
@@ -1,6 +1,5 @@
 --no-private
 --title=Google Cloud Language
---exclude lib/google/cloud/language/v1
 --exclude lib/google/cloud/language/v1.rb
```

Comment:
No, the namespaces are different in this package for some reason. There is no `lib/google/cloud/language/v1/doc/google/language/v1` path as in other packages, just `lib/google/cloud/language/v1/doc/google/cloud/language/v1`, so it needs to be included.